docs/reference/server-configuration.md

```diff
@@ -71,6 +71,11 @@ The connect worker configuration file, if spinning up {{ site.kconnect }}
 alongside the ksqlDB server. Don't set this property if you're using
 an external `ksql.connect.url`.
 
+## `ksql.extension.dir`
+
+The directory in which ksqlDB will look for UDFs. The default value
+is the `ext` directory relative to the ksqlDB's current working directory.
```

Comment:
```suggestion
is the `ext` directory relative to ksqlDB's current working directory.
```

Reply:
Is the working directory the same as `$CONFLUENT_HOME/bin`?

Reply:
I'm actually not sure. By default, it would actually be the directory that you execute the command to start ksqldb. But maybe our startup script change the CWD before launching.